plugins/chat/app/services/chat/search_chatable.rb

```diff
@@ -112,26 +86,55 @@ def search_users(context, guardian, contract)
       if context.excluded_memberships_channel_id
         user_search =
           user_search.where(
-            "NOT EXISTS (
-      SELECT 1
-      FROM user_chat_channel_memberships
-      WHERE user_chat_channel_memberships.user_id = users.id AND user_chat_channel_memberships.chat_channel_id = ?
-    )",
+            "NOT EXISTS (SELECT 1 FROM user_chat_channel_memberships WHERE user_id = users.id AND chat_channel_id = ?)",
```

Comment:
We don't need to repeat `user_chat_channel_memberships` inside the CTE. 

This makes it shorter and easier to inline.